src/components/LoginPage.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div class="loginComponent" >
+    <img alt="Vue logo" src="./../assets/nzpmclogo.jpg">
+    <ul>
+      <li>Username</li>
+      <input v-model="username" placeholder="">
+    </ul>
+    <ul>
+        <li>Password</li>
+      <input v-model="password" placeholder="" type="password">
+    </ul>
+    <div  class="btnContainer">
+    <button  v-on:click="proceed">Login</button>
+    </div>
+  </div>
+</template>
+
+
+<script>
+export default {
+  name: 'LoginPage',
+  methods: {
+      proceed() {
+          alert("You have now logged in")
+      }
+  }
+}
+</script>
+
+
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style scoped>
+img {
+    background-color: rosybrown;
+    width: 90%;
```

Comment:
```suggestion
    width: 100%;
```
Would be better to use padding instead of using 90%.